src/views/partials/doc-resource-classes.ejs

```diff
@@ -368,68 +337,105 @@
       <th align="left">Data Type</th>
     </tr>
   </thead>
-  <tbody></tbody>
+  <tbody>
+    <tr>
+      <td align="left">index</td>
+      <td align="left">The class index for shorthand searching.</td>
+      <td align="left">string</td>
+    </tr>
+
+    <tr>
+      <td align="left">class</td>
+      <td align="left">The class which this level refers to.</td>
+      <td align="left">
+        <a href="#namedapiresource">NamedAPIResource</a> (<a href="#class">Class</a>) Updated
+        ability scores, classes, and conditions
+      </td>
+    </tr>
+
+    <tr>
+      <td align="left">starting_equipment</td>
+      <td align="left">The starting equipment a character automatically gets</td>
+      <td align="left">
+        list <a href="#namedapiresource">NamedAPIResource</a> (<a href="#equipment">Equipment</a>)
+      </td>
+    </tr>
+
+    <tr>
+      <td align="left">starting_equipment_options</td>
+      <td align="left">
+        Starting equipment where the player must choose a certain number from the given list of
+        equipment
+      </td>
+      <td align="left">list <a href="#choice">Choice</a> (<a href="#equipment">Equipment</a>)</td>
+    </tr>
+
+    <tr>
+      <td align="left">url</td>
+      <td align="left">The URL reference of this resource.</td>
+      <td align="left">string</td>
+    </tr>
+  </tbody>
 </table>
 
 <h3>GET api/classes/{index}/spellcasting/</h3>
 
 <pre><code>{
+  "index": "bard",
   "class": {
-    "url": "/api/classes/bard",
-    "name": "Bard"
+    "url": "url/classes/bard",
```

Comment:
Should this change?

Reply:
it does seem a bit redundant

Reply:
Looks like it is `"url": "/api/classes/bard",` in the API?